### PR TITLE
Scaffold tool (create-component)

### LINKED ISSUE #128: Build task to scaffold a new component
The first steps in creating a new component are very similar, we could use some automation for that.
Among the tasks:

- create subdir with package.json, license and authors files
- a simple storybook story?

### PR DESCRIPTION
**Pull request by fczuardi**
- [x] creates a tool to generate the farmblocks scaffold as a npm package
- [x] closes #128

### CODE REVIEW

**Comment by fczuardi** [comment]
forgot to add the npm task, going back to [wip]

**Comment by fczuardi** [comment]
also, snyk failed with two security concerns, kinda of expected in the case of the shelljs dependency, that have the purpose of executing shell commands, but might need some digging on the current github discussions of each project (and might be possible to replace with less powerful tools) so I will keep this on hold for a little longer

**Comment by fczuardi** [comment]
I have opened issue #213 to keep track of the exceptions I am about to add in the snyk policy file, and to allow us to unblock this PR

**Comment by fczuardi** [comment]
ok, I am expecting travis to pass now, will unblock it for review and merge when we have time

File: packages/scaffold/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@crave/farmblocks-dev-scaffold",
+  "version": "0.1.0",
+  "description": "Helper tool for creating a new farmblock component",
+  "author": "Crave Food Services and AUTHORS",
```
Comment (alcferreira) [inline packages/scaffold/package.json]: Change to `Crave Food Systems and AUTHORS`

File: packages/scaffold/src/templates/package.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  description,
+  keywords = ["farmblocks"]
+}) =>
+  format(
+    `
+{
+  "name": "@crave/farmblocks-${shortName}",
+  "version": "0.1.0",
+  "description": "${description}",
+  "author": "Crave Food Services and AUTHORS",
```
Comment (alcferreira) [inline packages/scaffold/src/templates/package.js]: Change to `Crave Food Systems and AUTHORS`

**Comment by fczuardi** [comment]
change requests addressed

**Pull request merged**